group_vars/all.yml

```diff
@@ -1,7 +1,14 @@
 ---
+# MISC/Generic variables
+generic_tmp_dir: "{{ misc.misc06.path }}/tmp/"
+job_working_root_dir: "{{ jwd.jwd06.path }}" # we set this in gxconfig, but it should not be used, our object_store_conf is overwriting it
+object_store_cache_dir: "{{ cache.cache06.path }}/object_store_cache/"
```

Comment:
Where are those variables `{{ cache.cache06.path }}` defined?

Reply:
https://github.com/usegalaxy-eu/mounts/blob/85a8f18752b9d8eb401b7d622b5ca0e842457ac2/mountpoints.yml#L461-L464

We import the `mountpoints.yml` as a variable file in specific ansible playbooks